templates/scripts/golangci.yml.tpl

```diff
@@ -76,60 +21,153 @@ linters:
     - goconst
     - gocritic
     - gocyclo
-    - gofmt
-    - goimports
-    - revive
     - gosec
-    - gosimple
     - govet
     - ineffassign
     - lll
     # - misspell        # The reason we're disabling this right now is because it uses 1/2 of the memory of the run.
     - nakedret
+    - revive
     - staticcheck
-    - typecheck
     - unconvert
-    - unparam
-    - unused
+    # - unparam # Previously suppressed by --fast flag; disabled now to reduce noise
+    # - unused # Previously suppressed by --fast flag; disabled now to reduce noise
```

Comment:
```suggestion
    # - unparam # Previously suppressed by --fast in v1, disabling for now to avoid noise
    # - unused # Previously suppressed by --fast in v1, disabling for now to avoid noise
```